XpringKit/XRPPathElement.swift

```diff
@@ -1,7 +1,20 @@
 /// A path on the XRPLedger
 /// - SeeAlso: https://xrpl.org/paths.html#path-specifications
 public struct XRPPathElement: Equatable {
+
+  /// If present, this path step represents rippling through the specified address.
```

Comment:
It's odd that You prefix these with 'this field is optional' and you use '(optional)' on the third? Any reason to not use consistent formatting?

Reply:
Nope - I think that is a relic of grabbing comments directly from xrpl.org, in which some of them use an "(Optional)" at the beginning and others don't.  I updated a bunch of this in `xpring4j` while explicitly transforming fields to `Optional` types so I'll actually clean this up here real quick.